relay-event-normalization/src/normalize/mod.rs

```diff
@@ -219,22 +219,17 @@ fn normalize_app_start_spans(event: &mut Event) {
 /// The reasoning behind this normalization, is that the SDK sends namespace-agnostic metric
 /// identifiers in the form `metric_type:metric_name@metric_unit` and those identifiers need to be
 /// converted to MRIs in the form `metric_type:metric_namespace/metric_name@metric_unit`.
-fn normalize_metrics_summary_mris(value: &mut Value) {
-    let Value::Object(metrics) = value else {
-        return;
-    };
-
-    let metrics = mem::take(metrics)
+fn normalize_metrics_summary_mris(metrics_summary: &mut MetricsSummary) {
+    metrics_summary.0 = mem::take(&mut metrics_summary.0)
```

Comment:
What do you think if we add an accessor method to make the API friendlier to use? A `.value()` method in `MetricsSummary` that hides the `mem::take` would be friendlier to deal with. What do you think?

```rs
    metrics_summary.0 = metrics_summary.value().into_iter()...
```

Reply:
No strong opinions here, @Dav1dde what is your thought?